core/org.eclipse.birt.core.tests/test/org/eclipse/birt/core/archive/AllArchiveTests.java

```diff
@@ -57,7 +57,7 @@ public static Test suite( )
 		test.addTestSuite( FolderToArchiveTest.class );
 		test.addTestSuite( InputStreamRefreshTest.class );
 		test.addTestSuite( SpecialCharacterTest.class );
-		test.addTestSuite( TestBuffer.class );
+		// test.addTestSuite( TestBuffer.class );

```

Comment:
We shouldn't have a code that just commented out. Let's either remove it or better convert it to the TestCase